app/jobs/fastly_log_processor.rb

```diff
@@ -11,28 +11,28 @@ def initialize(bucket, key)
   end
 
   def perform
-    counts = download_counts
+    StatsD.increment('fastly_log_processor.processed')
 
-    # Temporary feature flag while we roll out fastly log processing
-    if ENV['FASTLY_LOG_PROCESSOR_ENABLED'] != 'true'
-      # Just log & exit w/out updating stats
-      Delayed::Worker.logger.info "Processed Fastly log counts: #{counts.inspect}"
-      StatsD.increment('fastly_log_processor.processed')
+    log_ticket = LogTicket.pop(key: key, directory: bucket)
+    if log_ticket.nil?
+      StatsD.increment('fastly_log_processor.extra')
```

Comment:
extra, means a processor that was enqueued but didnt have any ticket/log to process. That should not happen, unless we manually queue jobs.